### PR TITLE
Histogram unit tests

### PR DESCRIPTION
**Pull request by mrivi**
- add tests
- change `bin`, `dist` and `age` to float

### CODE REVIEW

**Comment by baumanta** [comment]
hey martina :) I'm now working on changing the whole histogram stuff to not use a binary one anymore. This will change quite some stuff, for one the histograms will be eigen matrices and also the whole determine freedirections and costmap things will probably change completely. This is fine with me, but I would not spend more time in cleaning up there, because soon it will change anyway

File: local_planner/src/nodes/histogram.cpp [hunk 1]
```diff
@@ -48,9 +49,9 @@ void Histogram::downsample() {
   resolution = 2 * resolution;
   z_dim = z_dim / 2;
   e_dim = e_dim / 2;
-  std::vector<std::vector<double> > temp_bin;
-  std::vector<std::vector<double> > temp_age;
-  std::vector<std::vector<double> > temp_dist;
+  std::vector<std::vector<float> > temp_bin;
```
[the remark this answers is not in the document]
Comment (mrivi) [reply local_planner/src/nodes/histogram.cpp]: No, the intermediate steps aren't binary. For example https://github.com/PX4/avoidance/blob/master/local_planner/src/nodes/planner_functions.cpp#L139-L140

**Comment by mrivi** [comment]
I wrote these tests before we had the call on Monday so I didn't know what you were working on. We can merge the tests and change them accordingly after the re-write of the histogram logic.

**Comment by baumanta** [comment]
sounds good!

File: local_planner/src/nodes/histogram.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (nicovanduijn) [reply local_planner/src/nodes/histogram.cpp]: Interesting. I definitely don't understand the logic in that file...

**Comment by mrivi** [comment]
not needed because of #195

**Pull request closed without merging**